doc/oper.xml

```diff
@@ -1964,3 +1964,162 @@ gap> ModularProduct(NullDigraph(0), CompleteDigraph(10));
 </Description>
 </ManSection>
 <#/GAPDoc>
+
+
+<#GAPDoc Label="StrongProduct">
+<ManSection>
+  <Oper Name="StrongProduct" Arg="D1, D2"/>
+  <Returns>A digraph.</Returns>
+  <Description>
+  If <A>D1</A> and <A>D2</A> are symmetric digraphs without multiple edges,
+  then <C>StrongProduct</C> calculates the <E>strong product digraph</E>
+  (<C>SPD</C>) of <A>D1</A>and <A>D2</A>.
+
+  <C>SPD</C> has vertex set <C>V1 x V2</C> where <C>V1</C> is the vertex set of
+  <A>D1</A> and <C>V2</C> is the vertex set of <A>D2</A> (a vertex
+  <C>[a, b]</C> has label <C>(a - 1) * |V2| + b</C> in the  output). There is
+  an edge from <C>[a, b]</C> to <C>[c, d]</C> when at least one of the
+  following three conditions are satisfied:
+  <List>
+    <Item>
+      The vertices <C>a</C> and <C>c</C> of <A>D1</A> are equal and there is
+      an edge from <C>b</C> to <C>d</C> in <A>D2</A>.
+    </Item>
+    <Item>
+      The vertices <C>b</C> and <C>d</C> of <A>D2</A> are equal and there is
+      an edge from <C>a</C> to <C>c</C> in <A>D1</A>.
+    </Item>
+    <Item>
+      There is an edge from <C>a</C> to <C>c</C> in <A>D1</A> and there is
+      an edge from <C>b</C> to <C>d</C> in <A>D2</A>.
+    </Item>
+  </List>
+
+  The SPD of two paths of lengths <C>m</C> and <C>n</C> is also the King's Graph
+  for an <C>m</C> by <C>n</C> board. 
+
+  <Example><![CDATA[
+gap> D1 := Digraph([[2], [1, 3, 4], [2, 5], [2, 5], [3, 4]]);
+<immutable digraph with 5 vertices, 10 edges>
+gap> D2 := Digraph([[2], [1, 3, 4], [2], [2]]);              
+<immutable digraph with 4 vertices, 6 edges>
+gap> StrongProduct(D1, D2);                                                    
+<immutable digraph with 20 vertices, 130 edges>
+gap> StrongProduct(DigraphSymmetricClosure(ChainDigraph(3)),
+DigraphSymmetricClosure(ChainDigraph(4)));
```

Comment:
Multi-line commands in manual examples need a `> ` at the start of subsequent lines, like in test files.
```suggestion
> DigraphSymmetricClosure(ChainDigraph(4)));
```